includes/wikia/services/sass/SassService.class.php

```diff
@@ -23,9 +24,9 @@
  * @author Władysław Bodzek <wladek@wikia-inc.com>
  *
  */
-class SassService extends WikiaObject {
```

Comment:
I want to have a private constructor here (docs say that static `newFrom...` functions should be used instead).